### DISCUSSION TITLE
Can't resize logo

### DISCUSSION THREAD

**Discussion by m1do-git** [fastfetch-cli/fastfetch#589] [Q&A] [answered]
Using an image as logo, I tried to set different logo width values in config file but logo size never change. Also it's very strange that I was able to change left and right padding but not top . I'm on macOS Monterey with iTerm2 and imagemagick installed. Please help !!!

**Comment by CarterLi** [comment]
Which image protocol did you use?

**Reply by CarterLi** [reply, marked as answer]
Try setting both width and height

**Reply by m1do-git** [reply]
In the config file I set `--logo-type iterm`. Also in iTerm2 Preferences/Advance/ I need to enable "Show inline image at retina resolution" for the logo (image) to fit. If not the it will be real big like this 
<img width="1081" alt="1" src="https://github.com/fastfetch-cli/fastfetch/assets/118599628/e46735c0-0641-4511-a233-0ffbf4eccdc5">

**Reply by CarterLi** [reply]
I never changed that. Wasn't it enabled by default?

**Reply by m1do-git** [reply]
I think I turn that off before for neofetch image to work. Changing logo height works but need to adjust quite many times to find the perfect ratio though. Thank you !!

**Reply by CarterLi** [reply]
Try `--logo-preserve-aspect-radio`

**Reply by m1do-git** [reply]
<img width="1111" alt="2" src="https://github.com/fastfetch-cli/fastfetch/assets/118599628/c446c565-e110-4161-b6e0-80e47e0603b9">
